policy_packs/aws/ssm/enforce_install_packages_on_instances_using_ssm_package_installer/variables.tf

```diff
@@ -1,9 +1,9 @@
 variable "region_resource" {
-  description = "Enter the Turbot Resource ARN for the AWS Region where SSM Documents will be placed. For example: 'arn:aws::us-east-1:999999999999'"
```

Comment:
Should the var names be `region_arn` and `ssm_document_role_arn`? Do we normally give the full type of expected value in var names for policy packs?